uiowa/uiowa
# Feature: uids_base section backgrounds and featured images

Work for #1501.

# How to Test 
`blt frontend`

- Add a section background image and verify that it is working with other paragraphs
<img width="1796" alt="Screen Shot 2020-05-08 at 2 55 09 PM" src="https://user-images.githubusercontent.com/1036433/81443964-f8b60d80-913b-11ea-9f3f-5c5e027fb628.png">

- Add a page featured image for a basic and verify that it is rendering as a hero component (http://localhost:3000/components/detail/hero-block--left)

- Verify that with featured image removed, the page title and breadcrumbs display as normal and there is no hero component markup in the html. 

<img width="791" alt="Screen Shot 2020-05-08 at 3 02 17 PM" src="https://user-images.githubusercontent.com/1036433/81444553-f43e2480-913c-11ea-9037-97ca00509df8.png">

- Add an article featured image and caption and verify that it is rendering as a hero component. Verify that caption is visible on hover. 

<img width="1728" alt="Screen Shot 2020-05-08 at 1 38 39 PM" src="https://user-images.githubusercontent.com/1036433/81437905-7cb6c800-9131-11ea-8749-60adff841a23.png">

- Add a page title hero component via layout builder with an image that contains a global caption and verify that caption appears on hover. 

<img width="1699" alt="Screen Shot 2020-05-08 at 1 37 48 PM" src="https://user-images.githubusercontent.com/1036433/81437889-77f21400-9131-11ea-8e53-f63f89a2806c.png">

- Verify that a very long page title will wrap within the hero component

<img width="1556" alt="Screen Shot 2020-05-08 at 1 38 43 PM" src="https://user-images.githubusercontent.com/1036433/81437862-71639c80-9131-11ea-824b-eedde9b6115d.png">


# Other misc tests

- Verify that tour z-index updates are working https://brand.local.drupal.uiowa.edu/?tour (It looks like tour z-index needs to be set around  `z-index: 600;` to get above admin toolbar. 

<img width="1074" alt="Screen Shot 2020-05-08 at 1 39 32 PM" src="https://user-images.githubusercontent.com/1036433/81437982-9d7f1d80-9131-11ea-8ce8-0c5731955852.png">

- Verify that Colorbox captions on https://brand.local.drupal.uiowa.edu/showcase appear within the colorbox container without overflowing.  You might need to add an image and caption to test this. 

<img width="1109" alt="Screen Shot 2020-05-08 at 1 47 47 PM" src="https://user-images.githubusercontent.com/1036433/81438531-8ee53600-9132-11ea-8fea-e3222d280e1a.png">

- Embed a small image into the card body on https://brand.local.drupal.uiowa.edu/templates/email-signatures and verify that it is not stretched full-width of the container. 

<img width="755" alt="Screen Shot 2020-05-08 at 2 54 18 PM" src="https://user-images.githubusercontent.com/1036433/81443875-d623f480-913b-11ea-8549-d5b2101de704.png">

- Verify that search and menu buttons are aligned when using the tertiary header with toggle menu and that there is no logo tab overlap when search is expanded during sticky scroll. 

<img width="1863" alt="Screen Shot 2020-05-08 at 2 58 12 PM" src="https://user-images.githubusercontent.com/1036433/81444329-914c8d80-913c-11ea-859b-6a0b5f18f0c9.png">

<img width="1849" alt="Screen Shot 2020-05-08 at 2 57 42 PM" src="https://user-images.githubusercontent.com/1036433/81444269-7b3ecd00-913c-11ea-9b5a-b26e89883051.png">

<img width="1869" alt="Screen Shot 2020-05-08 at 2 58 05 PM" src="https://user-images.githubusercontent.com/1036433/81444296-842f9e80-913c-11ea-856b-cf467b909d39.png">
